R/tab_create_modify.R

```diff
@@ -363,8 +399,28 @@ tab_row_group <- function(data,
   # Capture the `rows` expression
   row_expr <- rlang::enquo(rows)
 
-  # Create a row group if a `group` is provided
-  if (!is.null(group)) {
+  if (rlang::as_label(row_expr) != "NULL" && is.null(label)) {
+
+    stop(
+      "If specifying a row group for declared `rows`, a `label` must ",
+      "be provided.",
+      call. = FALSE
+    )
+  }
+
+  # Stop function if nothing was provided for `rows`
+  if (rlang::as_label(row_expr) == "NULL")  {
+
+    stop(
+      "No value was provided for `rows`. Please use either of:\n",
+      "* A vector of row captions provided in `c()`\n",
+      "* A vector of row indices\n",
+      "* A helper function focused on selections (e.g., `starts_with()`)",
+      call. = FALSE
+    )
+  }
```

Comment:
Let's add a unit test that does `expect_warning(tbl %>% tab_row_group(others = "blah"))`

Reply:
Now in.